storyblok/storyblok-design-system
# subtask(WORK-212): SbDataTable action changes

## Pull request type

Jira Link: [WORK-212](https://storyblok.atlassian.net/browse/WORK-212)




## How to test this PR



Go to SbDataTable > Actions Menu, select a row and see the toolbar with dynamic action buttons.

## What is the new behavior?

- It is now possible to dynamically set action buttons for SbDataTable. Each button emits the value defined within its object. Before, we had static buttons that did nothing.
- Uploaded 2 new icons: publish and unpublish
